en/advanced-topics/dynamic-atlas.md

```diff
@@ -10,7 +10,8 @@ Dynamic Atlas is according to the rendering order to determine whether the textu
 
 ## Enable, disable Dynamic Atlas
 
-During the initialization process of Cocos Creator, different [CLEANUP_IMAGE_CACHE](https://docs.cocos.com/creator/api/en/classes/macro.html#cleanupimagecache) value will be set according to different platforms. When `CLEANUP_IMAGE_CACHE` is disabled, Dynamic Atlas will be enabled by default.
+During the initialization process of Cocos Creator, different [CLEANUP_IMAGE_CACHE](https://docs.cocos.com/creator/api/en/classes/macro.html#cleanupimagecache) value will be set according to different platforms. When `CLEANUP_IMAGE_CACHE` is disabled, Dynamic Atlas will be enabled by default.<br>
+Enable dynamic atlas takes up additional memory, while different platforms consume different memory sizes. Dynamic atlas is currently disabled by default on Mini Games and Native platforms, but is recommended to open if your project still has extra memory space.
```

Comment:
Enabling
while the actual used memory space depends on the running platform.
and -> or
open -> activate it
extra -> enough